src/snapshot-io/index.ts

```diff
@@ -147,23 +147,22 @@ class SnapshotClient {
       })
       .filter(
         (space) =>
-          ((network === SupportedChainId.MAINNET.toString() &&
-            !space.private) ||
-            network !== SupportedChainId.MAINNET.toString()) &&
+          ((network === SupportedChainId.MAINNET && !space.private) ||
+            network !== SupportedChainId.MAINNET) &&
           (verified as any)[space.id] !== -1
       )
-      .filter((space) => space.network === network);
+      .filter((space) => space.network === network.toString());
     const list = orderBy(filters, ['followers', 'score'], ['desc', 'desc']);
-    this.spaces = list.map((item: any) => ({
-      id: item.id,
-      name: item.name,
-      avatar: item.avatarUri,
-      network: item.network,
-      admins: item.admins,
-      period: item.voting.period ? item.voting.period : undefined,
-      strategies: item.strategies,
-      followers: item.followers,
-    }));
+    this.spaces = list.map(
+      (item: any): SnapshotSpace => ({
```

Comment:
Nit: Try and avoid any in production code if you can.

Reply:
I want too, but snapshot sdk is javascript project, defined as any

Reply:
@deep-quality-dev add a comment above to mark this as intentional to the linter `eslint-disable-next-line @typescript-eslint/no-explicit-any`